report.py

```diff
@@ -1,21 +1,40 @@
-from telethon.tl.types import ChannelParticipantsAdmins
+from telethon.tl.types import (
+    ChannelParticipantsAdmins,
+    MessageEntityEmail,
+    MessageEntityMention,
+    MessageEntityMentionName,
+    MessageEntityTextUrl,
+    MessageEntityUrl
+)
 
 
 async def init(bot):
     # utils = modules['utils']
 
-    @bot.on(events.NewMessage(pattern='\/report'))
+    @bot.on(events.NewMessage(pattern='(?\/|#)report'))
     async def handler(event):
         # mentions = '<a href="tg://user?id={}">{}</a> '.format(
         #     event.sender_id,
         #     utils.get_display(event.sender)
         # )
-        if not event.reply_to_msg_id:
-            return
         reply_message = await event.get_reply_message()
+        if not reply_message:
+            return
+        is_link = any([
+            True if isinstance(
```

Comment:
`True if condition() else False` is the same as just `condition()`. Please change this.